src/server/src/main/java/io/cassandrareaper/resources/auth/RestPermissionsFilter.java

```diff
@@ -43,6 +44,17 @@ public boolean isAccessAllowed(ServletRequest request, ServletResponse response,
     if (isCorsEnabled() && RequestUtils.isOptionsRequest(request)) {
       return true;
     }
+
+    Subject subject = getSubject(request, response);
+
+    if (!subject.getPrincipals().getRealmNames().contains("jwtRealm")
+        && !RequestUtils.getSessionTimeout().isNegative()
+        && subject.getSession().getStartTimestamp().before(
```

Comment:
This is going to "timeout" now, even if there were legitimate requests that would otherwise keep things alive, is that what you're going for?

Reply:
Nope, no particular concern, just making sure I understood.  The session time is now a hard set number, no amount of application activity would keep it open and at some point the user would get logged out unexpectedly.  Not the experience you'd want, but I don't think that's a big deal for the way Reaper's UI is used though.

Reply:
Maybe we can make this an opt-in feature, so that the default behavior remains the same but users with stronger requirements can enable it?

Reply:
My guess is that changing the default to be more secure is fine.  The trade here in my mind is a slight usability downgrade (which I think is actually unlikely to really impact folks) for a security requirement that would serve to keep running the tool viable in more strict environments.